apps/site-nft/src/.htaccess

```diff
@@ -0,0 +1,52 @@
+<ifModule mod_rewrite.c>
+
+
+  #######################################################################
+  # GENERAL                                                             #
+  #######################################################################
+
+  # Make apache follow sym links to files
+  Options +FollowSymLinks
```

Comment:
Do we need FollowSymLinks? It could be a security risk.

Reply:
I don't know if it's possible here but we could also add ServerSignature at Off and ServerTokens Prod. If those are still called that way

Reply:
Ha!  That's where that error message comes from!

This .htaccess file is not actually used (it's not used on Firebase Hosting, which is the hosting we will almost certainly go with). 

I'll delete the whole file